source/features/follow-file-renames.tsx

```diff
@@ -13,7 +13,7 @@ async function findRename(
 	user: string,
 	repo: string,
 	lastCommitOnPage: string
-) {
+): Promise<Record<string, any>> {
```

Comment:
```suggestion
): Promise<AnyObject> {
```

Reply:
Done.

Reply:
Didn't do this one as I just realized I can commit your suggestion, but did the one below.